Documentation/ColumnsConfig/Type/User/Index.rst

```diff
@@ -155,7 +155,7 @@ dbType
 dbType = json example
 ~~~~~~~~~~~~~~~~~~~~~
 
-The system extension :t3ext:`reaction` uses a TCA field of type `user` with
+The system extension :t3ext:`webhooks` uses a TCA field of type `user` with
```

Comment:
Below is a code example which references EXT:reaction. Also the table sys_raction is referenced. Please check and correct the according places, if applicable.

Reply:
I think the reference to table `sys_reaction` is correct.